src/elisp/treemacs-themes.el

```diff
@@ -61,6 +61,7 @@
 (cl-defmacro treemacs-create-theme (name &key icon-directory extends config)
   "Create a new (bare) theme with the given NAME.
 - ICON-DIRECTORY is the (mandatory) theme's location.
+- EXTENDS is the theme to be extended.
```

Comment:
Commits need to be squashed, the commit message should be prefixed with `[Themes]`.

Only this change should remain its own commit since it's unrelated to the larger theme addition.

Reply:
Ok!